scripts/backport

```diff
@@ -23,7 +23,21 @@ shift 1
 TO_BRANCHES="$@"
 
 # Determine the remote to use
-git remote -v show | grep "https://github.com/[Dd]ata[Dd]og/dd-trace-py (push)" | read REMOTE _
+# DEV: If anything here fails, then default to 'origin'
+# DEV: We might have https://github.com/DataDog/dd-trace-py, or git@github.com:DataDog/dd-trace-py.
+if git remote -v show | grep -e "[Dd]ata[Dd]og/dd-trace-py\(\.git\)\? (push)" > /dev/null;
+then
+    REMOTE=$(git remote -v show | grep -e "[Dd]ata[Dd]og/dd-trace-py\(\.git\)\? (push)" | awk '{print $1}')
+else
+    echo "Failed to automatically determine the remote to use, falling back to 'origin'"
+    REMOTE="origin"
+fi
+
+if [[ -z "${REMOTE}" ]];
+then
+    echo "Failed to determine which remote to use, falling back to 'origin'"
+    REMOTE="origin"
+fi
```

Comment:
```suggestion
# Default to 'origin'
REMOTE="origin"

# DEV: If anything here fails, then default to 'origin'
# DEV: We might have https://github.com/DataDog/dd-trace-py, or git@github.com:DataDog/dd-trace-py.
if git remote -v show | grep -e "[Dd]ata[Dd]og/dd-trace-py\(\.git\)\? (push)" > /dev/null;
then
    REMOTE=$(git remote -v show | grep -e "[Dd]ata[Dd]og/dd-trace-py\(\.git\)\? (push)" | awk '{print $1}')
else
    echo "Failed to automatically determine the remote to use, falling back to 'origin'"
fi
```

This could work as well to ensure we don't have an unbound variable `REMOTE`.

We probably don't need the `-z` check ?